tox.ini

```diff
@@ -374,7 +374,7 @@ commands =
     mysql_contrib: pytest {posargs} tests/contrib/mysql
     mysqldb_contrib: pytest {posargs} tests/contrib/mysqldb
     pylibmc_contrib: pytest {posargs} tests/contrib/pylibmc
-    pylons_contrib: pytest {posargs} tests/contrib/pylons
+    pylons_contrib: python -m pytest {posargs} tests/contrib/pylons
```

Comment:
why do we need to make this change?

Reply:
I've made this change because `pylons` was complaining that it couldn't find `cache.conftest` inside `tests` on Python 2.7.

Reply:
I wonder why that is only an issue for Pylons?

Reply:
I wonder so too, but I couldn't get to the bottom of it 🙁

Reply:
Turns out `pylons` installs its own test suite in `site-package` under `tests` and that gets looked up before our `tests` package 🤦